### PR TITLE
test: call decorator story functions

### PR DESCRIPTION
**Pull request by jinlee93**
### Summary:
Slack context: https://czi-edu.slack.com/archives/CTFV79JH4/p1684258065353129
Calling `Story()` rather than using it as a React fn component seems to fix most of the "No Display Name" error when clicking "Show Code" in the Docs tab.
This is fine according to [SB docs](https://storybook.js.org/docs/react/writing-stories/decorators#snippet-your-component-with-decorator)

image
image

### Test Plan:

- [x] CI tests / new tests are not applicable
  - visual regression and jest tests pass

### CODE REVIEW

File: .storybook/recipes/MenuWithAvatarButton.stories.tsx [hunk 1]
```diff
@@ -13,41 +13,37 @@ export default {
...
-        <Menu.Item onClick={() => alert('Item clicked')}>
-          Trigger Action
-        </Menu.Item>
-        <Menu.Item disabled href="https://example.org/" icon="warning">
-          Not Possible (disabled)
-        </Menu.Item>
-      </Menu.Items>
-    </Menu>
-  ),
+  },
```
Comment (jinlee93) [inline .storybook/recipes/MenuWithAvatarButton.stories.tsx:46]: Also if args are not spread into the render function, "Show Code" breaks as well, so snuck the conversion of using `children` here

**Review by timzchang — APPROVED** [review]
Do we know what causes the issue?

**Review by jeremiah-clothier — APPROVED** [review]
Weird, I think there a few tickets related to 'No Display Name' that can be closed if this merges

**Comment by booc0mtaco** [comment]
> Do we know what causes the issue?

It's ... weird https://github.com/storybookjs/storybook/issues/12596#issuecomment-723440097

@jinlee93 I wonder if that `excludeDecorators` option fixes this w/o having to do `{Story()}`?

**Comment by jinlee93** [comment]
> > Do we know what causes the issue?
> 
> It's ... weird [storybookjs/storybook#12596 (comment)](https://github.com/storybookjs/storybook/issues/12596#issuecomment-723440097)
> 
> @jinlee93 I wonder if that `excludeDecorators` option fixes this w/o having to do `{Story()}`?

I tried the `excludeDecorators` but didn't work :(

**Comment by jinlee93** [comment]
> Do we know what causes the issue?

Didn't do too full of a deep dive but seems to be the storybook docs add-on related and how some properties are passed on via args

**Pull request merged**